src/content/architecture/globalnet/_index.en.md

```diff
@@ -23,30 +23,38 @@ This is largely problematic because most actual deployments use the default CIDR
 
 To support overlapping CIDRs in clusters connected through submariner, submariner has a component called Global Private Network, GlobalNet (`globalnet`). This GlobalNet is a virtual network specifically to support submariner's multi-cluster solution with a Global CIDR. Each cluster is given a subnet from this Global Private Network, configured as new cluster parameter `GlobalCIDR` (e.g. 169.254.0.0/16) which is configurable at time of deployment
 
-Once configured, each service and pod that requires cross-cluster access is allocated an IP, named `GlobalIp`, from this `GlobalCIDR` that is annotated on the Pod/Service object. This `GlobalIp` is used for all cross-cluster communication to and from to this Pod/Service. Routing and IPTable/OVS/OVN rules are configured to use this IP for ingress and egress. All address translations occur on the Gateway node.
+Once configured, each service and pod that requires cross-cluster access is allocated an IP, named `globalIp`, from this `GlobalCIDR` that is annotated on the Pod/Service object. This `globalIp` is used for all cross-cluster communication to and from to the Pod to globalIp of remote Service. Routing and IPTable/OVS/OVN rules are configured to use this IP for ingress and egress. All address translations occur on the Gateway node.
```

Comment:
```suggestion
Once configured, each Service and Pod that requires cross-cluster access is allocated an IP, named `globalIp`, from this `GlobalCIDR` that is annotated on the Pod/Service object. This globalIp is used for all cross-cluster communication to and from a Pod and the globalIp of a remote Service. Routing and IPTable/OVS/OVN rules are configured to use the globalIp for ingress and egress. All address translations occur on the Gateway node.
```

I think we need to be consistent with our use of case with pods/services. There's a mix of _pod_, _Pod_, and _POD_. We need to pick one. Looking at the k8s docs, they consistently use _Pod_ and _Service_.

Reply:
Yeah makes sense @tpantelis.